jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -75,9 +75,9 @@ def _on_data_added(self, msg):
             if len(msg.data.shape) == 3:
                 self.max_value = msg.data.shape[-1] - 1
                 self._watch_viewer(msg.viewer, True)
-            else:
-                # then the viewer is showing a static 2d image instead of cube data
-                self._watch_viewer(msg.viewer, False)
+            # else:
+            #     # then the viewer is showing a static 2d image instead of cube data
+            #     self._watch_viewer(msg.viewer, False)
```

Comment:
This is from a separate bug fix and will be removed before this gets out of draft. There was a problem where if 2D data was loaded into a viewer with 3D data, then the 2D data was removed, the 3D data would be stuck at one slice.